apps/jobs/views.py

```diff
@@ -266,8 +262,23 @@ def leaderboard(request, challenge_phase_split_id):
                                                 distinct_sorted_leaderboard_data,
                                                 request,
                                                 pagination_class=StandardResultSetPagination())
-    response_data = result_page
-    return paginator.get_paginated_response(response_data)
+
+    challenge_host_user = is_user_a_host_of_challenge(request.user, challenge_obj.pk)
+
+    # Check if the user is challenge host or not
```

Comment:
This comment is useless. Instead write something like `Show the Private leaderboard only if the user is a challenge host`